subsys/event_manager/em.ld

```diff
@@ -1,4 +1,15 @@
-SECTION_DATA_PROLOGUE(event_manager,,)
+event_types_all : ALIGN_WITH_INPUT
 {
-	KEEP(*("event_manager"));
-} GROUP_DATA_LINK_IN(ROMABLE_REGION, ROMABLE_REGION)
+	__start_event_types_all = .;
+	KEEP(*(SORT(event_types_*)));
+	__stop_event_types_all = .;
+} > FLASH
+
+ITERABLE_SECTION_ROM(event_listener, 8)
```

Comment:
why 8?

Reply:
its a size of struct - and yes sizeof(event_listener) will be better

Reply:
It does not have to be a size of struct. Alignment of struct is usually alignement of the member that has most strict requirement of it. It cannot be less but rarely it should be more.
I changed it to 4. I don't expect anything in the struct will need alignement of 8. 4 is sufficient